README.md

```diff
@@ -584,6 +584,68 @@ For more information about inheriting configuration from a gem please check
 * Avoid the usage of class (`@@`) variables due to their "nasty" behavior in
   inheritance.
 
+* Avoid the usage of class instance variables unless you really need them
```

Comment:
```suggestion
* Avoid the usage of instance variables on singletons unless you really need them
```

Reply:
Thanks for the suggestion, I slightly altered the entire text to incorporate it.